docs/essentials/skip-building-some-commits-with-ci-skip.md

```diff
@@ -9,7 +9,4 @@ trigger a build, just write `[ci skip]` or `[skip ci]` somewhere in your
 commit's message.
 
 !!! info "Usage with squashed commits"
-	When squashing commits before merging in another branch all commit messages
-will be merged as well. If a previous commit in the current branch contains
-`[ci skip]` or `[skip ci]` it will be propagated and the merged commit wil
-not build.
+	When squashing commits before merging in another branch all commit messages will be merged as well. If a previous commit in the current branch contains `[ci skip]` or `[skip ci]` it will be propagated and the merged commit will not build.
```

Comment:
```suggestion
	When squashing commits before merging in another branch, all commit messages will be merged as well. If a previous commit in the current branch contains `[ci skip]` or `[skip ci]`, it will be propagated, and the merged commit will not build.
```

Changes proposed by Grammarly ^